src/monitor.ts

```diff
@@ -115,11 +115,15 @@ const innerMonitor = <Callable, Scope extends string, Method extends string>({
 
 export const createMonitor =
   <Scope extends string>({ scope, ...initOptions }: InitOptions<Scope>) =>
-  <Callable, Method extends string>(method: CamelCase<Method>, callable: () => Callable, options?: MonitorOptions<Callable>) =>
+  <Callable, Method extends string>(
+    method: CamelCase<Method> extends never ? string : CamelCase<Method>,
```

Comment:
extends never ? : string
mind explaning?
CamelCase<Method> didn't let you put a variable as a scope / method name?

Reply:
Yeah, it resolves to never, so this is the workaround

Reply:
Also, look at the test I added, it failed before